packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -3101,6 +3125,54 @@ export interface RaiseHandCallFeature {
     raisedHands: RaisedHandState[];
 }
 
+// @public
+export type Reaction = {
+    reactionType: string;
+    receivedOn: Date;
+};
+
+// @public
+export interface ReactionButtonProps extends ControlBarButtonProps {
+    onReactionClick: (reaction: string) => Promise<void>;
+    reactionResources: ReactionResources;
+    strings?: Partial<ReactionButtonStrings>;
+}
+
+// @public
+export interface ReactionButtonStrings {
+    applauseReactionTooltipContent?: string;
+    ariaLabel: string;
+    heartReactionTooltipContent?: string;
+    label: string;
+    laughReactionTooltipContent?: string;
+    likeReactionTooltipContent?: string;
+    surprisedReactionTooltipContent?: string;
+    tooltipContent?: string;
+    tooltipDisabledContent?: string;
+}
+
+// @public
+export interface ReactionResources {
+    applauseReaction?: ReactionSprite;
+    heartReaction?: ReactionSprite;
+    laughReaction?: ReactionSprite;
+    likeReaction?: ReactionSprite;
+    surprisedReaction?: ReactionSprite;
+}
+
+// @public
+export type ReactionSprite = {
+    url: string;
+    frameCount: number;
+    size?: number;
```

Comment:
Just re-looking at this, should we make it:
```
frameSize?: {
  height: number;
  width: number
}
```

@Mohtasim 

(@dmceachernmsft don't block PR on this, we can push a change afterwards (but before stable release) if we want this last minute change)